### ISSUE TITLE
Cannot reload plugins in NVDA snapshots

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#14408]
### Steps to reproduce:
1. Install some add-ons an ensure they work.
2. Press NVDA+control+f3 or use the reload plugins option available under NVDA's tools menu.
3. Try to use add-ons.
### Actual behavior:

Plugins are terminated, not reloaded. Note that, when pressing shift+control+q after reloading plugins,, message about eMule version is not reported.

shift+ctrl+q
Versión 0.50a de eMule lista
ctrl+NVDA+f3
Plugins reloaded
shift+ctrl+q

### Expected behavior:
Plugins should be reloaded, not just terminated.

shift+ctrl+q
Versión 0.50a de eMule lista
ctrl+NVDA+f3
Plugins reloaded
shift+ctrl+q
Versión 0.50a de eMule lista

[nvda.log](https://github.com/nvaccess/nvda/files/10097680/nvda.log)

#### NVDA installed/portable/running from source:
Running from source

#### NVDA version:
source-master-85efa6d (2023.1.0.0)
#### Windows version:
Versión	10.0.22621 compilación 22621

#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
Windows in Spanish
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
nvda_snapshot_alpha-27190,85efa6d7 portable, with same behavior.
#### If NVDA add-ons are disabled, is your problem still occurring?
No, since the problem is related to add-ons.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by nvdaes** [comment]
This has been reported by other users.
See this `thread on the add-onsmailing list](https://nvda-addons.groups.io/g/nvda-addons/topic/95253295#19876)

**Comment by josephsl** [comment]
Hi,

Er, @LeonardDer, is this possibly a regression from the work on add-on directories?

Thanks.

**Comment by leonardder** [comment]
It could be. I'll have a look tomorrow.
